app/components/primer/alpha/overlay.html.erb

```diff
@@ -0,0 +1,30 @@
+<%= show_trigger_button %>
```

Comment:
`show_trigger_button` feels like a method that returns `true` or `false`. Can we call it something like `trigger_button` instead?